tests/test_gradient.py

```diff
@@ -602,6 +603,10 @@ def test_grad_constant(self):
                 + str(g_one)
             )
 
+    def test_verify_grad_no_rng(self):
```

Comment:
No need for the test, the changes in the code are okay by themselves

Reply:
I’ve removed the test as suggested. Let me know if there’s anything else to address!